### ISSUE TITLE
Feature: Get all members of groups in Facebook

### ISSUE THREAD

**Issue by thovt93** [thovt93/fbspider#2]

**Comment by thovt93** [comment]
@EricTurf Welcome

**Comment by EricTurf** [comment]
Seems like the API endpoint to get members from a group was deprecated earlier this year

https://developers.facebook.com/docs/graph-api/reference/v3.1/group/members

**Comment by thovt93** [comment]
@EricTurf  Exactly, so I do not use `graph-api`. I use `puppeteer` to do that. You can check `package.json`
